src/libcalamaresui/ViewManager.cpp

```diff
@@ -530,12 +540,12 @@ ViewManager::updateCancelEnabled( bool enabled )
 }
 
 void
-ViewManager::updateBackAndNextVisibility( bool visible)
+ViewManager::updateBackAndNextVisibility( bool visible )
 {
     UPDATE_BUTTON_PROPERTY( backAndNextVisible, visible )
 }
 
-QVariant
+QVariant 
```

Comment:
.. and -1 for trailing spaces here. If you run `ci/calamaresstyle` on the files you edit, it will massage things into the "right" shape; that includes things like include-file-orderinging.